rasr/feature_scorer.py

```diff
@@ -126,25 +127,53 @@ def __init__(self, prior_mixtures, scale=1.0, priori_scale=0.0, prior_file=None)
         self.config.normalize_mixture_weights = False
 
 
-class TFLabelContextFeatureScorer(FeatureScorer):
+class OnnxFeatureScorer(FeatureScorer):
     def __init__(
         self,
-        fs_tf_config,
-        contextPriorFile,
-        diphonePriorFile,
-        prior_mixtures,
-        prior_scale,
+        *,
+        mixtures,
+        model,
+        io_map,
+        label_log_posterior_scale=1.0,
+        label_prior_scale=0.7,
+        label_log_prior_file=None,
+        apply_log_on_output=False,
+        negate_output=True,
+        intra_op_threads=1,
+        inter_op_threads=1,
+        **kwargs,
     ):
-        super().__init__()
+        """
+        :param str mixtures: path to a *.mix file e.g. output of either EstimateMixturesJob or CreateDummyMixturesJob
+        :param str model: path of a model e.g. output of ExportPyTorchModelToOnnxJob
+        :param dict io_map: mapping between internal rasr identifiers and the model related input/output. Default key values
+                            are "features" and "output", and optionally "features-size"
```

Comment:
```suggestion
                            are "features" and "output", and optionally "features-size",
                            e.g. io_map = {"features": "data", "output": "classes"}
```

Reply:
I am not sure about the spacing here, maybe one or two spaces for the new line missing.